.circleci/copy-junit-reports.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+const {execSync} = require('child_process');
+const {join} = require('path');
+const {mkdirSync, rmSync, statSync, readdirSync, copyFileSync} = require('fs');
+
+/**
+ * Discover all test results, which @bazel/jasmine stores as `test.xml` files, in the directory and
+ * return back the list of absolute file paths.
+ */
+const findTestResultsInDir = function (dirPath, files) {
+  for (const file of readdirSync(dirPath)) {
+    const filePath = join(dirPath, file);
+    if (statSync(filePath).isDirectory()) {
+      files = findTestResultsInDir(filePath, files);
+    } else {
+      if (file === 'test.xml') {
+        files.push(filePath);
+      }
+    }
+  }
+  return files;
+};
+
+/** StdOut result from the bazel info command. */
+const bazelInfoOutput = execSync('yarn bazel info', {stdio: 'pipe', encoding: 'utf8'});
```

Comment:
bazel info bazel-testlogs
```suggestion
const bazelInfoOutput = execSync('yarn bazel info bazel-testlogs', {stdio: 'pipe', encoding: 'utf8'});
```

Reply:
I googled this for like 10 minutes and could not find how to do this.  I feel silly now.